### DISCUSSION TITLE
WYSIWYG

### DISCUSSION THREAD

**Discussion by danfoxley** [Zettlr/Zettlr#4748] [Q&A] [answered]
Please help me confirm, WYSIWYG options. I'd like to get more of a Typora feel, is there more to do in the default Zettlr install to get WYSIWYG exposed more?

I see WYSIWYG referenced as module that is/was? required to be turned on. I couldn't find advanced mode and a console to run the command mentioned in [this Reddit post.](https://www.reddit.com/r/Zettlr/comments/g17y4k/is_there_a_way_to_autohide_the_markdown_syntax/)

I see the [features](https://www.zettlr.com/features) say WYSIWYG, is an option.

Thanks!

**Comment by nathanlesage** [comment, marked as answer]
You can get WYSIWYG by enabling all renderers in the display preferences.
